owncloudApp/src/androidTest/java/com/owncloud/android/utils/matchers/BottomSheetFragmentItemViewMatchers.kt

```diff
@@ -0,0 +1,77 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Abel García de Prada
+ *
+ * Copyright (C) 2020 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ * <p>
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * <p>
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.owncloud.android.utils.matchers
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.annotation.ColorRes
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.matcher.BoundedMatcher
+import androidx.test.espresso.matcher.RootMatchers
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import com.owncloud.android.R
+import com.owncloud.android.presentation.ui.common.BottomSheetFragmentItemView
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+
+fun Int.bsfItemWithTitle(@StringRes title: Int, @ColorRes tintColor: Int?) {
```

Comment:
Yes, that is the key. The extension will be called by an Int. Any layout id could call it, so i decided to rename it in a specific way. It will avoid confusions if someone tries to use withTitle and the id does not match with a bottom sheet fragment item.